### PR TITLE
Fixed the duplicate files. 

### PR DESCRIPTION
**Pull request by YDutchy**
Note: Please do empty the recentGFA.txt :)

### CODE REVIEW

**Comment by YDutchy** [comment]
Iwan's comment should be fixed now

File: src/main/java/programminglife/controller/RecentFileController.java [hunk 1]
```diff
@@ -208,19 +222,24 @@ private void doesFileExist() {
...
         file4 = file3;
         file3 = file2;
         file2 = file1;
-        file1 = file.getAbsolutePath();
+        file1 = file;
+        System.out.println(file1);
+        System.out.println(file2);
+        System.out.println(file3);
+        System.out.println(file4);
+        System.out.println(file5);
```
Comment (MeAmAnUsername) [inline src/main/java/programminglife/controller/RecentFileController.java]: Please remove these sysouts

**Comment by YDutchy** [comment]
@toinehartman The error was first a pop-up message, forgot to edit it so that it prints a line into the console

**Comment by YDutchy** [comment]
Will definitely do what Iwan recommended if there is time later.

**Pull request merged**